assortment__darkmodejs.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "@assortment/darkmodejs" {
+  export = assortment__darkmodejs;

```

Comment:
What's the reasoning this is `assortment__darkmodejs ` instead of `assortment_darkmodejs`

Reply:
From our quick chat:

```
It's a convention TypeScript uses for packages with a vendor - usually it's just the name of the package but if you're using a vendor like assortment then it uses the double underscore to split the vendor and the package name - here's an example from Google
```